test/blocks/gnav/mocks/autocomplete..json

```diff
@@ -0,0 +1,6 @@
+{
+  "query_prefix": "",
+  "locale": "fr_FR",
+  "suggested_completions": [],
+  "elastic_search_time": 1.0
+}
```

Comment:
Nit: newline missing at the end of the file. Also, the filename seems to have a superfluous `.`, not sure if it's needed.